lib/dependencies/all.js

```diff
@@ -54,7 +48,6 @@ shaka.dependencies = class {
  * @enum {string}
  */
 shaka.dependencies.Allowed = {
-  muxjs: 'muxjs',
```

Comment:
I think this could break an app that tries to inject its own mux.js with .add(), which will throw when this is removed.  Maybe leave this for now, or add a check for `'muxjs'` in add() that calls Deprecated() and returns?

To be clear: it's great that we're dropping mux and won't use it.  But ignoring mux.js in the environment and deleting `muxjs_transmuxer.js` is backward compatible.  Removing it from `Allowed` without some other check in `add()` could cause an exception that wasn't there before, and so isn't backward compatible for some minority apps.

Reply:
Changed!